Arcade/FlxPongApi/Project.xml

```diff
@@ -80,7 +80,7 @@
 	<!--Place custom nodes like icons here (higher priority to override the HaxeFlixel icon)-->
 	<icon path="assets/HaxeFlixel_Addons.svg" />
 	
-	<assets path="assets/sounds" if="flash">
+	<assets path="assets/sounds">
```

Comment:
Shouldn't these be `if="web"`? I thought native targets don't support mp3.

Reply:
Right now I can't build iOs native to check this out but it looks like [iOs can play mp3](https://www.oreilly.com/library/view/ios-game-development/9781449369644/ch04.html) but can't .ogg
But in that case, I believe that on the absence of mp3 they would fail on compile time instead of runtime as it is on html5.